src/content/learn_ip_packet_for_beginner_by_kato.md

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
@imony could you change file name to `src/content/learn_ip_packets_for_beginners.md` ?

Reply:
`src/content/img/posts/learn_ip_packet_for_beginner_by_kato/title.png` here too